source/PlayerInfo.cpp

```diff
@@ -729,6 +716,10 @@ Ship *PlayerInfo::Flagship()
 }
 
 
+const shared_ptr<Ship> &PlayerInfo::FlagshipPtr() const
```

Comment:
Nope, remove this. Use `PlayerInfo::Flagship` for const access to the player's flagship.